### ISSUE TITLE
defineComponent is not defined error in Nuxt

### ISSUE THREAD

**Issue by toniengelhardt** [inokawa/virtua#438]
**Describe the bug**

I cannot get it to work in Nuxt (v3).

I'm getting a "defineComponent is not defined" error for both of these two scenarios:

1. Registering the component via plugin:
```ts
import { VList } from 'virtua/vue'

export default defineNuxtPlugin({
  name: 'virtua',
  parallel: false,
  async setup() {
    nuxtApp.vueApp.component('VList', VList)
  },
})
```
2. Simply importing the component and using it:
```html
<script setup lang="ts">
import { VList } from 'virtua/vue'
</script>

<template>
  <div>
    <VList :data="data" #default="item">
      <div>
	    {{ item.property }}
      </div>
	</VList>
  </div>
</template>
``` 

How to use the package with Nuxt?

**To Reproduce**

See description.

**Expected behavior**

The component should just work as shown in the example.

**Platform:**

- OS: MacOS
- Browser: Chrome
- Version of this package: 0.30.2
- Version of framework: vue 3.4.21, nuxt 3.11.2

**Additional context**

--

**Comment by inokawa** [comment]
Hi, does your config have `typescriptBundlerResolution: true`?
https://github.com/inokawa/virtua/issues/366#issuecomment-1934203872

**Comment by inokawa** [comment]
Oh, published bundle for vue seems to be broken for unknown reason. Could you try `virtua: "0.29.2"`?

**Comment by inokawa** [comment]
Fixed in `0.30.3`. Thanks!

**Comment by toniengelhardt** [comment]
Amazing, works like a charm 🙏🏽

Love the project!
